hibernate-core/src/test/java/org/hibernate/orm/test/orphan/onetomany/Product.java

```diff
@@ -19,7 +20,7 @@
 @Table(name="product")
 public class Product {
 	@Id
-	@GeneratedValue(strategy= GenerationType.AUTO)
+	@GeneratedValue(strategy= GenerationType.IDENTITY)
```

Comment:
Its generally frowned-upon to change the mapping of entities used in existing tests to match the needs of your new test(s).  If you need a specific mapping, create a model with that mapping